drivers/clock_control/clock_control_mcux_pcc.c

```diff
@@ -67,7 +67,12 @@ static const struct clock_control_driver_api mcux_pcc_api = {
 	.get_rate = mcux_pcc_get_rate,
 };
 
-#if DT_HAS_DRV_INST(0)
+#define CHECK_INSTANCE_NUMBER(inst) \
+	BUILD_ASSERT((inst) == 0, "unsupported temp instance");
```

Comment:
What's the thinking here?

Reply:
Just trying to move quickly. The driver is single-instance and was hard-coded to instance number 0, so if this blows up, the devicetree is violating the "there's just one instance" assumption and this driver needs to be looked at again.